distributions/standalone/pom.xml

```diff
@@ -38,6 +38,33 @@
                 </executions>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-plugins</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <skip>${skipDistributions}</skip>
+                            <useBaseVersion>true</useBaseVersion>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.mulesoft.weave</groupId>
+                                    <artifactId>mule-plugin-weave</artifactId>
+                                    <version>${weaveVersion}</version>
+                                    <classifier>dist</classifier>
+                                    <type>zip</type>
+                                </artifactItem>
```

Comment:
Instead of specifying the artifact here, can't you use the `copy-dependencies` goal? It will make more sense if we want to see what are the distribution dependencies with `dependency:tree`